.changelog/20715.txt

```diff
@@ -0,0 +1,6 @@
+```release-note:feature
+dns: queries now default to a refactored DNS server that is v1 and v2 Catalog compatible. 
```

Comment:
Do multi-line changelog entries do the right thing? I don't think I've authored one before.

Reply:
It should wrap like normal markdown, which I think is OK here.

Reply:
I also confirmed looking at 20812.txt - it should up as a single line 👍